### ISSUE TITLE
Problems with cache of old version

### ISSUE THREAD

**Issue by svrnwnsch** [Rebilly/ReDoc#806]
I deployed a new version of my api documentation. When i tested it locally with speccy everything worked. After deploying it on my website I get the error that some `$ref` link could not be resolved. After looking into the issue I see that the problem are some old caches. The problem here is I cannot update them either by reloading on firefox with `strg+f5` nor by manually deleting all website data and cookies.

The only workaround I found is to open the website in a private window, where everything works fine.

Information: I use the docker container tagged latest (Release 2.0.0-rc.2)

**Comment by RomanGotsiy** [comment]
> After looking into the issue I see that the problem are some old caches.
why do you think so? any details?

> I get the error that some `$ref` link could not be resolved.
and if you try to load that link, does it work?

**Comment by svrnwnsch** [comment]
I did not try to load the link directly but as it worked in private mode, the only explanation I have is that some caching does not work all the time. But as there is a NGINX proxy and a htaccess file to limit access there could be other caching mechanics not related to redoc which caused the failure. Additionally I noticed that I did not need to enter any authentication when reloading the page. So it could be also some access limitations problems.
